### DISCUSSION TITLE
Volume buttons don't affect bluetooth headphones.

### DISCUSSION THREAD

**Discussion by moshpirit** [i3/i3#5940] [Q&A]
I am running bluetoothd and use blueberry to connect the headphones. The problem is that Vol± buttons don't affect the headphones, but they work fine with the speakers. I'm using these scripts (I think they are the default with Endeavor OS):
- [volume.sh](https://github.com/panos21sk/EndeavourOsDotfiles/blob/master/i3/scripts/volume)
- [volume_brightness.sh](https://github.com/panos21sk/EndeavourOsDotfiles/blob/master/i3/scripts/volume_brightness.sh)

I am not a developer, but they look good, I guess.  Am I missing anything?

**Comment by okraits** [comment]
I don't know which volume buttons you mean and your links don't work.
But in general, you need to select the correct sink.

See: https://wiki.archlinux.org/title/PulseAudio#Keyboard_volume_control

**Reply by moshpirit** [reply]
Hi! I rechecked the links and they seem to work fine. 

I mean `XF86AudioRaiseVolume` and `XF86AudioLowerVolume` buttons.

I've tried `sh -c "pactl set-sink-mute 0 false ; pactl set-sink-volume 0 +5%"` in the terminal but it doesn't seem to work (I checked that I have `tactl`  installed)

I've connected the bluetooth and there's no changes.

```
pactl list sinks short
0       alsa_output.pci-0000_00_1f.3-platform-skl_hda_dsp_generic.HiFi__HDMI3__sink     module-alsa-card.c      s16le 2ch 48000Hz       SUSPENDED
1       alsa_output.pci-0000_00_1f.3-platform-skl_hda_dsp_generic.HiFi__HDMI2__sink     module-alsa-card.c      s16le 2ch 48000Hz       SUSPENDED
2       alsa_output.pci-0000_00_1f.3-platform-skl_hda_dsp_generic.HiFi__HDMI1__sink     module-alsa-card.c      s16le 2ch 48000Hz       SUSPENDED
3       alsa_output.pci-0000_00_1f.3-platform-skl_hda_dsp_generic.HiFi__Speaker__sink   module-alsa-card.c      s16le 2ch 48000Hz       SUSPENDED
4       bluez_sink.41_42_AA_67_A4_49.handsfree_head_unit        module-bluez5-device.c  s16le 1ch 16000Hz       RUNNING

```

**Reply by orestisfl** [reply]
Try opening the links on a private tab, they are not opening for me either

**Reply by moshpirit** [reply]
You're right! sorry:

- [volume.sh](https://paste.fo/raw/7b933acc570a)
- [volume_brightness.sh](https://paste.fo/raw/e0ee786a0062)

**Reply by orestisfl** [reply]
> `sh -c "pactl set-sink-mute 0 false ; pactl set-sink-volume 0 +5%"`

From the sink list above, it seems like you need to use sink 4 instead of 0
